.changeset/mighty-hairs-train.md

```diff
@@ -0,0 +1,61 @@
+---
+"@comet/admin": minor
+---
+
+Add a more actions menu helper component
+
+This is done to provide a simple api to create a more actions menu for a list of items. 
+
+**Example:**
+
+```tsx
+<MoreActionsMenu
+    selectionSize={selectionSize}
+>
+    {({ handleClose }) => (
+        <>
+            <MenuItem
+                onClick={() => {
+                    handleMoveClick();
+                    handleClose();
```

Comment:
Shouldn't all items close per default (same as in RowActions)?

Reply:
Not sure... If we don't enforce it, it's more flexible. Whats your opinion @thomasdax98 ?

Reply:
I also think the behavior should be consistent, e.g. with `RotActions`.